### PR TITLE
chore: update TimeController title

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue
Part of the typography fixes.

## Description
Updates the TimeController title to it doesn't line wrap and updates the syncTranslation script to take type into account.

### TODO:

- [x] Update FR
- [x] Update DE
- [x] Update ES
- [x] Update IT
- [x] Update SV
- [x] Update DA

### Double check

- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
Waiting for a second look from @Alportan and possibly some Romanian translations as well. Will merge it after that 👍🏻

File: web/src/locales/ro.json [hunk 1]
```diff
@@ -43,6 +43,18 @@
...
     "now": "Acum"
   },
+  "time-controller": {
+    "title": {
+      "hourly": "NOT CORRECT ",
+      "daily": "Daily data for:",
+      "monthly": "Monthly data for:",
+      "yearly": "Yearly data for:"
+    },
+    "daily": "daily",
```
Comment (Alportan) [inline web/src/locales/ro.json:53]:
```suggestion
    "daily": "pe zi",
```

File: web/src/locales/ro.json [hunk 2]
```diff
@@ -43,6 +43,18 @@
...
+    "hourly": "hourly",
```
Comment (Alportan) [inline web/src/locales/ro.json:54]:
```suggestion
    "hourly": "pe oră",
```

File: web/src/locales/ro.json [hunk 3]
```diff
@@ -43,6 +43,18 @@
...
+    "monthly": "monthly",
```
Comment (Alportan) [inline web/src/locales/ro.json:55]:
```suggestion
    "monthly": "pe luna",
```

File: web/src/locales/ro.json [hunk 4]
```diff
@@ -43,6 +43,18 @@
...
+    "yearly": "yearly"
```
Comment (Alportan) [inline web/src/locales/ro.json:56]:
```suggestion
    "yearly": "pe an"
```

File: web/src/locales/ro.json [hunk 5]
```diff
@@ -43,6 +43,18 @@
     "noParserInfo": "Încă nu avem informații despre această zonă.<br/> Ai vrea să ajuți? Poți contribui prin <a href=\"{{link}}\" target=\"_blank\">date pentru zonă</a>.",
     "now": "Acum"
   },
+  "time-controller": {
+    "title": {
+      "hourly": "NOT CORRECT ",
```
Comment (Alportan) [inline web/src/locales/ro.json:48]:
```suggestion
      "hourly": "Date orare pentru:",
```

File: web/src/locales/ro.json [hunk 6]
```diff
@@ -43,6 +43,18 @@
...
+      "daily": "Daily data for:",
```
Comment (Alportan) [inline web/src/locales/ro.json:49]:
```suggestion
      "daily": "Date zilnice pentru:",
```

File: web/src/locales/ro.json [hunk 7]
```diff
@@ -43,6 +43,18 @@
...
+      "monthly": "Monthly data for:",
```
Comment (Alportan) [inline web/src/locales/ro.json:50]:
```suggestion
      "monthly": "Date lunare pentru:",
```

File: web/src/locales/ro.json [hunk 8]
```diff
@@ -43,6 +43,18 @@
...
+      "yearly": "Yearly data for:"
```
Comment (Alportan) [inline web/src/locales/ro.json:51]:
```suggestion
      "yearly": "Date anuale pentru:"
```

**Review by Alportan — CHANGES_REQUESTED** [review]
Updated the Romanian keys 🙌

**Review by Alportan — APPROVED** [review]
Looking good, let's do this! 🌟

**Pull request merged**